### ISSUE TITLE
Remote x session crashes xquartz

### ISSUE THREAD

**Issue by odinsbane** [XQuartz/XQuartz#106]
It started with one of the release candidates. I don't know the number off hand. I upgraded to 2.8.0_rc2, as was suggested when I started xquartz. That didn't fix anything. I still would get a random crash. I tried down grading to 2.7.11 and now I cannot start an x forwarding session unless I change the DISPLAY variable. 

Yes I have logged out since installing 2.7.11 again.

I don't know how I ended up on the beta channel, that's probably where the bug began because everything was fine. Now I cannot start a functional remote session and I don't know what to do.

**Comment by jeremyhu** [comment]
Yeah, 2.7.11 unfortunately released, pointing users to the beta channel.  I had no idea about that until putting out the first beta in over 4 years ... =/  That issue should never happen again with the new build system because the channel is now based on the version string instead of needing to provide both at build time.

As for re-installing 2.7.11, a restart won't fix the situation because the 2.8.0 remnants are still around.  You'd need to uninstall 2.8.0 as described at https://www.xquartz.org/FAQs.html.

As for your primary reason for needing to downgrade (I still would get a random crash), can you please provide the crash log or any other useful information to traige that problem?

**Comment by odinsbane** [comment]
Great thanks for the response. I didn't realize I had to remove 2.8.0 first. The instructions to remove 2.7 and earlier seem to have a typo. "To uninstall XQuartz 2.8 and later, execute the following in Terminal:"

**Comment by jeremyhu** [comment]
Thanks, fixed the typo in fac3fae

**Comment by odinsbane** [comment]
I removed both 2.7 and 2.8, then re-installed 2.8. I haven't been getting crashes. It was happening pretty consistently before so maybe uninstalling resolved some issue.

**Comment by jeremyhu** [comment]
Ok, well then unfortunately, there's nothing here to go on. I'll close this out, but if you're able to reproduce and provide a crash log, I can take a look.
